item-spec/item-spec.md

```diff
@@ -215,6 +216,105 @@ Deprecation notice: GeoTiff previously used the media type `image/vnd.stac.geoti
+                    "name": "B04",
+                    "common_name": "red"
+                },
+                {
+                    "full_width_half_max": 0.045,
+                    "center_wavelength": 0.56,
+                    "name": "B03",
+                    "common_name": "green"
+                },
+                {
+                    "full_width_half_max": 0.098,
+                    "center_wavelength": 0.4966,
+                    "name": "B02",
+                    "common_name": "blue"
+                }
+            ],
+            "href": "s3://sentinel-s2-l2a/tiles/30/P/ZQ/2019/7/13/0/R10m/TCI.jp2"
+        },
+        "B05": {
+            "title": "Band 5",
+            "type": "image/jp2",
+            "roles": [
+                "data"
+            ],
+            "gsd": 20,
+            "eo:bands": [
+                {
+                    "full_width_half_max": 0.019,
+                    "center_wavelength": 0.7039,
+                    "name": "B05"
+                }
+            ],
+            "href": "s3://sentinel-s2-l2a/tiles/30/P/ZQ/2019/7/13/0/R20m/B05.jp2"
+        },
+        "B08": {
+            "title": "Band 8 (nir)",
+            "type": "image/jp2",
+            "roles": [
+                "data"
+            ],
+            "eo:bands": [
+                {
+                    "full_width_half_max": 0.145,
+                    "center_wavelength": 0.8351,
+                    "name": "B08",
+                    "common_name": "nir"
+                },
+            ],
+            "href": "s3://sentinel-s2-l2a/tiles/30/P/ZQ/2019/7/13/0/R10m/B08.jp2"
+        }
+    }
+}
+```
+
+The Sentinel-2 overall `gsd` is 10m, because this is the best spatial resolution among all the bands and is defined in Item properties so it can be searched on. However, Band 5 has a `gsd` of 20m, so that asset specifies the `gsd` as well, which overrides the Item `gsd` for this one asset. Reduced resolution versions of files could also be included as assets and the `gsd` correectly represented.
+
+For `eo:bands`, it could be put in Item properties as an array of all the bands, but in this case it's not. Instead, the assets each define an array containing the spectral band information for that asset (in the order the bands appear in the file).
+
+##### Common Use Cases for Item Properties in Assets
+
+Overriding or providing Item Properties only in the Assets makes discovery more difficult and should generally be avoided. However, there are some core and extension fields for which providing them at at the Asset level can prove to be very useful for using the data.
+
+- `gsd`: Specify some assets with different spatial resolution than the overall best resolution.
+- `updated`: Provide individual timestamps on when an asset was last updated. Since the Item property only tells when any of the Item was updated, this can provide additional info for users to determine which asset(s) might have been updated.
```

Comment:
Updated is for metadata only and can't be used for such a use-case, unfortunately.

Reply:
What I think is more useful could be a datetime for timeseries.

Reply:
My thought for updated is that the link might be changed, in which case you have updated the metadata 

I'll add datetime

Reply:
Well, I think updated on assets would be quite confusing. People would think it's the time for the data, but the spec clearly says it's the metadata. Also, wouldn't updated in the item properties change also? Related: https://github.com/radiantearth/stac-spec/pull/769#issuecomment-621494830 - Maybe we need updated for data...

Reply:
Yes, if you updated an asset URL you'd update the Item properties.   But imagine that all the assets have an `updated` field, you'd be able to tell what the last one was that was updated.
Anyway, I'll remove it from this list.

Reply:
I think there can never be multiple updated fields as it refers to the date the STAC metadata file was written. You'd need a separate field that is the asset update time.